Framework/MDAlgorithms/src/IntegrateEllipsoids.cpp

```diff
@@ -301,29 +245,8 @@ void IntegrateEllipsoids::init() {
                   "If this options is enabled, then the the top 1% of the "
                   "background will be removed"
                   "before the background subtraction.");
-
-  declareProperty("SatelliteRegionRadius", .1, mustBePositive,
-                  "Only events at most this distance from a peak will be "
-                  "considered when integrating");
-
-  declareProperty("SatellitePeakSize", .08, mustBePositive,
-                  "Half-length of major axis for satellite peak ellipsoid");
-
-  declareProperty("SatelliteBackgroundInnerSize", .08, mustBePositive,
-                  "Half-length of major axis for inner ellipsoidal surface of "
-                  "satellite background region");
-
-  declareProperty("SatelliteBackgroundOuterSize", .09, mustBePositive,
-                  "Half-length of major axis for outer ellipsoidal surface of "
-                  "satellite background region");
-
-  declareProperty("GetUBFromPeaksWorkspace", false,
```

Comment:
This algorithm is used only in ORNL and we already had a conversation with the instrument scientists. They decided they wanted a rewrite of the algorithm instead of a new version. Plus I got the impression that the satellite options were more a hindrance than an advantage to them :smile: